<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubSceneChooseDialog</class>
 <widget class="QDialog" name="SubSceneChooseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Subscene</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="radioExisting">
         <property name="text">
          <string>Choose a Subscene</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listExistingScenes">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>false</bool>
       </property>
       <property name="layoutMode">
        <enum>QListView::Batched</enum>
       </property>
       <property name="gridSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
       <property name="batchSize">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolAddSubScene">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolRemoveSubScene">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Publish</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Browse Online</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QRadioButton" name="radioNew">
     <property name="text">
      <string>Create a new Subscene</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelNameOfScene">
     <property name="text">
      <string>Name of the Scene</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineNameOfScene">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SubSceneChooseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SubSceneChooseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listExistingScenes</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>SubSceneChooseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioExisting</sender>
   <signal>toggled(bool)</signal>
   <receiver>listExistingScenes</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioExisting</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineNameOfScene</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioNew</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineNameOfScene</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioNew</sender>
   <signal>toggled(bool)</signal>
   <receiver>listExistingScenes</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioExisting</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolAddSubScene</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioExisting</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolRemoveSubScene</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioNew</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolAddSubScene</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioNew</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolRemoveSubScene</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
